src/OSPSuite.Core/Domain/Mappers/MoleculeBuilderToMoleculeAmountMapper.cs

```diff
@@ -82,7 +81,6 @@ public MoleculeAmount MapFrom(MoleculeBuilder moleculeBuilder, IContainer target
 
          allLocalParameters.Each(x => moleculeAmount.Add(_parameterMapper.MapFrom(x, simulationBuilder)));
 
-         _keywordReplacerTask.ReplaceIn(moleculeAmount);
```

Comment:
this is done as part of the molecule replacement in the model so not sure why this is here